python/spacewalk/satellite_tools/reposync.py

```diff
@@ -1192,19 +1192,27 @@ def _populate_erratum(self, notice):
             updated_date = self._to_db_date(notice["updated"])
         else:
             updated_date = self._to_db_date(notice["issued"])
+
+        existing_packages = existing_errata["packages"] if existing_errata else []
+        npkgs = [pkg for c in notice["pkglist"] for pkg in c["packages"]]
+
+        e = importLib.Erratum()
+        e["packages"] = self._updates_process_packages(
+            npkgs, e["advisory_name"], list(existing_packages)
```

Comment:
Isn't `existing_packages` always a list?

Reply:
Oh, I didn't consider that you wanted to create a copy. In that case I suggest to use `existing_packages.copy()`, it's more explicit.